src/api/app/models/review.rb

```diff
@@ -5,6 +5,8 @@ class NotFoundError < APIException
     setup 'review_not_found', 404, 'Review not found'
   end
 
+  VALID_REVIEW_STATES = [:new, :declined, :accepted, :superseded, :obsoleted].freeze
```

Comment:
why did you add freeze here and not in the other one? :thinking:

Reply:
Seems I forgot that. I'll fix it now.